docker/thrift/Dockerfile

```diff
@@ -0,0 +1,90 @@
+# global args
+ARG SPARK_VERSION=2.4.5
+ARG HADOOP_VERSION=2.7.7
+ARG HADOOP_MINOR_VERSION=2.7
```

Comment:
Rather than have to declare two args and having to keep both in sync, what about calculating `HADOOP_MINOR_VERSION` from `HADOOP_VERSION`?

I'm not an expert at bash substitution by any means, but I believe this does the trick:

```
# Get 2-part minor version string (e.g. `2.7.7` -> `2.7`)
ENV HADOOP_MINOR_VERSION=${HADOOP_VERSION%.*}
```

Reply:
```
Step 4/26 : ARG HADOOP_MINOR_VERSION=${HADOOP_VERSION%.*}
ERROR: Service 'thrift' failed to build: failed to process "${HADOOP_VERSION%.*}": missing ':' in substitution
```

I don't think this is supported by Docker, curious if/how you made it work